src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -139,5 +122,10 @@ public override Task OnDisconnectedAsync(Connection connection)
                 }
             }
         }
+
+        private static string GetInvocationId()
+        {
+            return Guid.NewGuid().ToString("N");
```

Comment:
Yeah, we can do that actually. The initial concern was in situations where we were invoking across multiple connections and wanted to generate the ID only once and send the same message to each connection. Originally, @davidfowl and I thought we'd need to use GUIDs because each connection would be on a different Invocation ID, but I was confusing Result messages and Invocation messages :). In either case, we can just generate a seemingly-random integer (using a per-endpoint incrementing int like this) and it'll be fine.

I do still want to keep it a string (which I changed when switching to GUID) because I think it's better to be flexible (and since we don't care about them as numbers, except as a source of uniqueness)